core/plugins/org.fusesource.ide.camel.model.service.core/META-INF/MANIFEST.MF

```diff
@@ -29,7 +29,8 @@ Require-Bundle: org.eclipse.core.resources;bundle-version="3.10.0",
  com.fasterxml.jackson.core.jackson-databind;bundle-version="2.5.0",
  org.fusesource.ide.foundation.core,
  org.fusesource.ide.foundation.ui,
- org.eclipse.m2e.core.ui;bundle-version="1.6.2"
+ org.eclipse.m2e.core.ui;bundle-version="1.6.2",
+ org.jboss.tools.common.jaxb;resolution:=optional
```

Comment:
using resolution optional means that with Java 11, if org.jboss.tools.common.jaxb bundle is not available it won't work.
When inside Jboss Tools, the plugin is part of it i think. I don't think that it is the case for vanilla Fuse Tooling. I think we need to add the dependency to this plugin in one of our Camel feature explicitely.

Reply:
Remove the optional one

Reply:
sure that optional can be removed? i think that we will have 2 jaxb implementation when on Java 8. This is something that is not working as far as I know.